src/renderer/views/About/About.js

```diff
@@ -13,6 +13,7 @@ export default Vue.extend({
     'ft-button': FtButton
   },
   data: function () {
+    document.title = `${this.$t(this.$route.meta.title)} - ${process.env.PRODUCT_NAME}`
```

Comment:
Changing this.$route.meta.title to "About.About" and repeating for all other cases should fix the window not being properly updated when back button is pressed. Thoughts?